### PR TITLE
basic edit pass

### PR DESCRIPTION
**Pull request by davidchesnut**
@waldekmastykarz great article! I'm submitting this PR with my edit pass. I will also add some comments to this PR about some of the changes I made. There were a couple sentences that were not working grammatically and I attempted to fix them. so please review and be sure the technical meaning still works for any changes I made. if good please merge back into your original PR #285. I will also have a few more review comments there that I'll add.

Thanks!

### CODE REVIEW

File: docs/spfx/web-parts/guidance/build-custom-property-pane-controls.md [hunk 1]
```diff
@@ -174,34 +174,33 @@ In the web browser, add the **List items** web part to the canvas and open its p
...
+The SharePoint Framework offers you a standard dropdown control that allows users to select a specific value. The dropdown control is built in a way that requires all its values to be known upfront. If you want to load the values dynamically or you're loading values asynchronously from an external service and you don't want to block the whole web part, building a custom dropdown control is a viable option.
 
-> **Note:** In the drop 6 of the SharePoint Framework there is a bug in the Office UI Fabric React Dropdown component that causes the control built in this tutorial to work incorrectly. A temporary workaround is to edit the **node_modules/@microsoft/office-ui-fabric-react-bundle/dist/office-ui-fabric-react.bundle.js** file and change line **12027** from:
-
-```js
-isDisabled: this.props.isDisabled !== undefined ? this.props.isDisabled : this.props.disabled
-```
+When creating a custom property pane control that uses React in the SharePoint Framework, the web part consists of a class that registers the control with the web part, and a React component that renders the dropdown and manages its data.
```
Comment (waldekmastykarz) [inline docs/spfx/web-parts/guidance/build-custom-property-pane-controls.md]: This is not quite correct. It's the custom property pane control not the web part that consists of a class that registers the control with the web part and a component for rendering the control.

File: docs/spfx/web-parts/guidance/build-custom-property-pane-controls.md [hunk 2]
```diff
@@ -142,7 +142,7 @@ export default class ListItems extends React.Component<IListItemsProps, {}> {
                 Welcome to SharePoint!
               </span>
               <p className='ms-font-l ms-fontColor-white'>
-                Customize SharePoint experiences using Web Parts.
+                Customize SharePoint experiences using web parts.
```
Comment (waldekmastykarz) [inline docs/spfx/web-parts/guidance/build-custom-property-pane-controls.md:145]: This is the standard string that is created by the SharePoint Framework Yeoman generator.

File: docs/spfx/web-parts/guidance/build-custom-property-pane-controls.md [hunk 3]
```diff
@@ -309,21 +308,21 @@ export default class AsyncDropdown extends React.Component<IAsyncDropdownProps,
...
+The **AsyncDropdown** class represents the React component used to render the asynchronous dropdown property pane control. When the component first loads, the **componentDidMount** method, or its **disabled** or **stateKey** properties will change, and it will load the available options by calling the **loadOptions** method passed through the properties. Once the options are loaded the component updates its state showing the available options. The dropdown itself is rendered using the [Office UI Fabric React dropdown component](http://dev.office.com/fabric#/components/dropdown). When the component is loading the available options it displays a spinner using the [Office UI Fabric React spinner component](http://dev.office.com/fabric#/components/spinner).
```
Comment (waldekmastykarz) [inline docs/spfx/web-parts/guidance/build-custom-property-pane-controls.md:311]: Without parenthesis surrounding _the componentDidMount method_ the sentence seems confusing. It was meant as a pointer to the right fragment in code but if it obstructs reading then we could leave it out.

**Review by waldekmastykarz — CHANGES_REQUESTED** [review]
Hi David,

Thank you for your feedback and changes. They make it easier to read the article.
I've added a few comments regarding your changes. Could you please have a look at them?

Thank you,

Waldek

**Comment by davidchesnut** [comment]
One of your comments at line 179 disappeared after I made a change to address it. Anyway the line now reads: When creating a custom property pane control that uses React in the SharePoint Framework, the control consists of a class that registers the control with the web part, and a React component that renders the dropdown and manages its data.

**Comment by davidchesnut** [comment]
BTW: If there is a more efficient way to do these reviews I'm all ears. My first time trying this editing thing in GitHub. :-)

**Comment by waldekmastykarz** [comment]
So is mine ;) Not sure if we're doing things correctly though so it might be that.

**Comment by waldekmastykarz** [comment]
@davidchesnut PR is merged and directly pushed into my PR to the SharePoint repo. We can pick it up from there

**Pull request merged**